### PR TITLE
Add GRUB installation steps to managing-vm-kernels.md

### PR DESCRIPTION
**Pull request by SaswatPadhi**
Resolves QubesOS/qubes-issues#6178

Added GRUB installation steps:
- `grub2-install` on Fedora
- `grub-install` on Debian

### CODE REVIEW

File: user/advanced-topics/managing-vm-kernels.md [hunk 1]
```diff
@@ -287,7 +293,11 @@ Install distribution kernel image, kernel headers and the grub.
 sudo apt install linux-image-amd64 linux-headers-amd64 grub2 qubes-kernel-vm-support
 ~~~
 
-If you are doing that on a qube based on "Debian Minimal" template, a grub gui will popup during the installation, asking you where you want to install the grub loader. You must select /dev/xvda (check the box using the space bar, and validate your choice with "Enter".)
+If you are doing that on a qube based on "Debian Minimal" template, a grub gui will popup during the installation, asking you where you want to install the grub loader. You must select /dev/xvda (check the box using the space bar, and validate your choice with "Enter".) If this popup does not appear during the installation, you must manually setup `grub2` by running:
+
+~~~
+sudo grub-install /dev/xvda
+~~~
```
Comment (andrewdavidwong) [inline user/advanced-topics/managing-vm-kernels.md:300]: If I'm reading this correctly, this repeats the command that you instructed the user to run above. Won't that be confusing? Also, should we really be telling the user the run this command above, at the start, when we later mention the GUI option?

**Review by andrewdavidwong — CHANGES_REQUESTED** [review]

**Comment by andrewdavidwong** [comment]
Never mind. I was looking at a collapsed diff. Deleting my erroneous comments.

**Comment by SaswatPadhi** [comment]
@marmarek, please take a look when you get a chance. Fairly small change, and I've tested the proposed change on multiple TemplateVMs.

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of f4c00061dd0cec5d596e1c7e815e26008105d5d3
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmIFGQgACgkQ8yiUvpaE
k4pdwxAA0nQ7y+rmflgfM1QbxHIComUmMs+zhoBIXpKgnBZYKjIjQV9RrQLljoYd
FI2n5CSkhSQkHh6Tpf0HwtqTUBDr+tZeIOtvD4ZCYVQwxepb+vrC4e7To88HXQiS
D3/QbaLS/gGm8L+8dZ2142ls3mdy+NFjUPLtUUXVq0auj3+rVDFFBRt9HSnM6jFf
8CREmKNvRt1IiIYFPwI15skGYBIu/LeuYU9/AK1N77aprrgET7nF18yuB8Xvllnw
lUCTX6+niggEOL7cg4KTy12fSrxt33k+trjsjkU11EUKwOWCEP03x2EjlhN5SHjR
0oFC7Y2tB2MaOEK8s8ypgkr5k6akYek5Vw41j5kytzCrSAQjm5zlHnx8lKI3zjVs
NpmxXhRKDah3QfUHlpKm7tRfzO3dFYkxZKFrFE8a97I/15r9gJrQE9S7A04aTCi+
jV3GCjmhfyYnIqp4CAfAktCkFf92qTvU12D7Ruk+52k15LbZavrNqNl6JyEjH5mt
GXMPWN9jlzUFsDMnNycnxefgZNPkt9wO4nu+/6UBW01R8s/eCcFnsZrSHWIFuNbn
i5ozezehh46b057mVC/4sXbrl6QsxiBTZnMHDWpejF+Ne5x9+Gig32+IDeK8iFMj
RxKBxQ0BG/6YcgAkVnigyy9vjArqduWirUW3N7/z3ytiAlcxNKw=
=WZV/
-----END PGP SIGNATURE-----
```

**Pull request merged**